feat: upgrade version for react dsfr

Viczei opened the pull request:
closes #6628

Viczei committed:
feat: upgrade version for react dsfr

maxgfr commented on packages/code-du-travail-frontend/src/modules/config/StartDsfrLight.tsx:
```diff
@@ -1,15 +1,20 @@
 import Link from "../common/Link";
 
-declare module "@codegouvfr/react-dsfr/next-appdir" {
+declare module "@codegouvfr/react-dsfr/spa" {
```
t'es sur ? on est pas une `spa`

Viczei replied:
ah oui c'est vrai, bizarre que j'ai due changé ca pour que ca passe avec la nouvelle version. j'avais une erreur qui bloquait avec next-appdir

maxgfr commented on packages/code-du-travail-frontend/src/modules/config/StartDsfrLight.tsx:
```diff
@@ -1,15 +1,20 @@
+startReactDsfr({
+  defaultColorScheme: "light",
+  Link,
+  nonce: "true",
```
c'est pas un boolean ici ?

Viczei committed:
remove spa

Viczei committed:
update snap

maxgfr commented on packages/code-du-travail-frontend/src/modules/common/__tests__/__snapshots__/DisplayContent.test.tsx.snap:
```diff
@@ -645,11 +645,11 @@ exports[`DisplayContent Infographic should replace div with infographic class to
           >
             Format PDF - 3200Ko
-          </p>
+          </div>
```
ça faut qu'on le corrige car c'est du texte

maxgfr commented on packages/code-du-travail-frontend/src/modules/home/__tests__/__snapshots__/index.test.tsx.snap:
```diff
@@ -235,11 +235,11 @@ exports[`<Home /> should match snapshot 1`] = `
                     >
                       Tool description
-                    </p>
+                    </div>
```
là aussi à corriger, et à voir pour tous les tiles

maxgfr reviewed (commented):
c'est vraiment utile les snapshot pour ce genre d'upgrade ! On devra en faire plus en vrai

maxgfr reviewed (commented):
c'est vraiment utile les snapshot pour ce genre d'upgrade ! On devra en faire plus en vrai

m-maillot committed:
fix tile issue

# Conflicts:
#	packages/code-du-travail-frontend/src/modules/outils/page-principale/ToolTile.tsx

# Conflicts:
#	packages/code-du-travail-frontend/src/modules/fiche-service-public/builder/__tests__/__snapshots__/FicheServicePublic.test.tsx.snap
#	packages/code-du-travail-frontend/src/modules/fiche-service-public/builder/components/__tests__/__snapshots__/Tabulator.test.tsx.snap

m-maillot committed:
fix: conflicts

m-maillot committed:
fix: conflicts

m-maillot committed:
fix: conflicts
